src/main/java/com/alipay/oceanbase/rpc/exception/ObTableGlobalIndexRouteException.java

```diff
@@ -0,0 +1,69 @@
+/*-
+ * #%L
+ * OBKV Table Client Framework
+ * %%
+ * Copyright (C) 2021 OceanBase
+ * %%
+ * OBKV Table Client Framework is licensed under Mulan PSL v2.
+ * You can use this software according to the terms and conditions of the Mulan PSL v2.
+ * You may obtain a copy of Mulan PSL v2 at:
+ *          http://license.coscl.org.cn/MulanPSL2
+ * THIS SOFTWARE IS PROVIDED ON AN "AS IS" BASIS, WITHOUT WARRANTIES OF ANY KIND,
+ * EITHER EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO NON-INFRINGEMENT,
+ * MERCHANTABILITY OR FIT FOR A PARTICULAR PURPOSE.
+ * See the Mulan PSL v2 for more details.
+ * #L%
+ */
+
+package com.alipay.oceanbase.rpc.exception;
+
+public class ObTableGlobalIndexRouteException extends ObTableException {
+
+    /*
+     * Ob table global index route exception.
+     */
+    public ObTableGlobalIndexRouteException() {
+    }
+
+    /*
+     * Ob table global index route exception.
+     */
+    public ObTableGlobalIndexRouteException(int errorCode) {
+        super(errorCode);
+    }
+
+    /*
+     * Ob table global index route exception.
+     */
+    public ObTableGlobalIndexRouteException(String message, int errorCode) {
+        super(message, errorCode);
+    }
+
+    /*
+     * Ob table global index route exception.
+     */
+    public ObTableGlobalIndexRouteException(String message) {
+        super(message);
+    }
+
+    /*
+     * Ob table global index route exception.
+     */
+    public ObTableGlobalIndexRouteException(String message, Throwable cause) {
+        super(message, cause);
+    }
+
+    /*
+     * Ob table global index route exception.
+     */
+    public ObTableGlobalIndexRouteException(Throwable cause) {
+        super(cause);
+    }
+
+    /*
+     * Is need refresh table entry.
+     */
+    public boolean isNeedRefreshTableEntry() {
+        return false;
```

Comment:
Could you make sure that OB_ERR_KV_GLOBAL_INDEX_ROUTE does not need to refresh the route? What could users do after receiving this exception to fix this problem?

Reply:
If someone meet this exception, it is most likely a bug and need to fix it. We cannot get the correct route after refreshing router because the indexTableName is high likely primary table name. But firstly let us catch this exception and to find the reason. If receive this problem, user can do retry in its implement.